examples/counter/src/counter.rs

```diff
@@ -52,6 +54,19 @@ pub enum Evt {
     Decreased(u64),
 }
 
+pub trait CounterExt {
```

Comment:
In these examples I'd rather not intoduce sugar, I'd prefer to show the bare bone API.

Reply:
Ok